### ISSUE TITLE
False positive: EXTENSION_USED_BUT_NOT_FOUND

### ISSUE THREAD

**Issue by bjornblissing** [CesiumGS/3d-tiles-validator#335]
The following minimal example will emit the `EXTENSION_USED_BUT_NOT_FOUND` warning:

```json
{
  "asset": {
    "version": "1.1"
  },
  "geometricError": 0,
  "root": {
    "boundingVolume": {
      "region": [
        -1.31968,
        0.69887,
        -1.31960,
        0.69895,
        0.0,
        100.0
      ]
    },
    "geometricError": 0,
    "refine": "ADD",
    "content": {
      "uri": "unlit_tile.glb"
    }
  },
  "extensionsUsed": ["3DTILES_content_gltf"],
  "extensions": {
    "3DTILES_content_gltf": {
      "extensionsUsed": [
        "KHR_materials_unlit"
      ]
    }
  }
}
```

Log from validator:
```json
{
  "date": "2025-05-28T13:35:36.333Z",
  "numErrors": 0,
  "numWarnings": 1,
  "numInfos": 0,
  "issues": [
    {
      "type": "EXTENSION_USED_BUT_NOT_FOUND",
      "path": "/extensionsUsed",
      "message": "The extension '3DTILES_content_gltf' was declared in 'extensionsUsed', but not found",
      "severity": "WARNING"
    }
  ]
}
```
